Packages/com.unity.inputsystem/InputSystem/Plugins/DualShock/DualShockGamepad.cs

```diff
@@ -9,9 +9,9 @@
 namespace UnityEngine.InputSystem.DualShock
 {
     /// <summary>
-    /// A Sony DualShock controller.
+    /// A Sony DualShock/DualSense controller.
     /// </summary>
-    [InputControlLayout(displayName = "PS4 Controller")]
+    [InputControlLayout(displayName = "Playstation Controller")]
```

Comment:
Hah, thank you! Stumbled over the same thing today and thought "WTF is there no PlayStation controller!"

Ah, should read `PlayStation`, not `Playstation`.

Reply:
fixed